superset-frontend/src/explore/components/controls/DatasourceControl.jsx

```diff
@@ -148,11 +160,11 @@ class DatasourceControl extends React.PureComponent {
                   {t('Explore in SQL Lab')}
                 </MenuItem>
               )}
-              {!!this.props.onDatasourceSave && (
+              {this.props.isEditable ? (
```

Comment:
is `this.props.isEditable && (<MenuItem>)` prettier?

Reply:
Maybe. I think either is fine since Prettier doesn't complain.

Reply:
I like the piece of mind ternary gives me as it's more explicit and wouldn't break things when `isEditable` is a truthy non-boolean value.